test/cep_client_test.exs

```diff
@@ -32,6 +38,18 @@ defmodule CepClientTest do
       assert error == :error
       assert reason == "Unavailable."
     end
+
+    test "should accept a module as sources keyword" do
+      {:ok, address} = Cep.Client.get_address("00000-001", sources: [:dummy, CepSourceTest])
+
+      assert address.city == "Piraque"
+    end
+
+    test "should accept a module as source keyword" do
```

Comment:
I don't feel like this test is useful. The test above is already covering the same functionality. What do you think? I don't mind keeping it though, as long as the same is not exactly the same and the other one you are adding. 😄

Reply:
I think it's two cases becouse sources accepts a list of sources, I had to ensure using `source` keyword accepts module as parameter.